pkg/commands/copy.go

```diff
@@ -59,11 +59,15 @@ func (c *CopyCommand) ExecuteCommand(config *v1.Config, buildArgs *dockerfile.Bu
 		if err != nil {
 			return err
 		}
+		if fi.IsDir() && !strings.HasSuffix(fullPath, string(os.PathSeparator)) {
```

Comment:
This should probably use `filepath.IsAbs(fullPath)`

Reply:
It is suffix, not prefix. I put this code in here to easily check if source is a directory.

Reply:
woops, yup I see that mistake now. I guess I'm slightly confused about this check. So it is `true` for a fi that is a directory but whose `fullPath` does not end in a `/` ?

Is the goal of this check to make sure `fullPath` is formatted as expected?

Reply:
The goal of this code is what you say. I thought it would be inefficient to call `s.Lstat()` or` os.Stat()` every time when I pass src directly to a function.